gm4_chairs/data/gm4_chairs/functions/main.mcfunction

```diff
@@ -1,7 +1,7 @@
-#saddle item on a stair block -> spawn pig
-execute as @e[type=minecraft:item,nbt={Item:{id:"minecraft:saddle"}},tag=!gm4_chairs_ignore] at @s if block ~ ~ ~ #minecraft:stairs[half=bottom] positioned ~ ~-.1 ~ align xyz unless entity @e[type=minecraft:pig,tag=gm4_chairs,dx=0,dz=0,dy=0] at @s run function gm4_chairs:create_chair
+# detect saddle item on a stairs and spawn chair
+execute as @e[type=minecraft:item,nbt={Item:{id:"minecraft:saddle"}},tag=!gm4_chairs_ignore] at @s if block ~ ~ ~ #minecraft:stairs[half=bottom] align xyz positioned ~.5 ~ ~.5 unless entity @e[type=minecraft:pig,tag=gm4_chairs,distance=..0.4] run function gm4_chairs:create_chair
```

Comment:
The previous implementation of this line made sure there is no chair already within the same block as this chair.
Your new implementation checks that there is no chair in a 0.4 block radius around the center of the bottom face of this block. Is this intended? See the attached image for a visualization.
![Selection](https://user-images.githubusercontent.com/41843855/140376950-8b36828a-9530-4ebc-9874-630b6983f0e9.png)
_Fig. 1) Old implementation in blue, new implementation in red._